pandas/io/formats/style.py

```diff
@@ -895,7 +895,8 @@ def background_gradient(self, cmap='PuBu', low=0, high=0, axis=0,
         low, high : float
             compress the range by these values.
         axis : int or str
-            1 or 'columns' for columnwise, 0 or 'index' for rowwise
+            0 or 'index' for columnwise, 1 or 'columns' for rowwise, or
+            ``None`` for tablewise
```

Comment:
Our linter may complain about the lack of a period after tablewise. LGTM otherwise though.

Reply:
There's actually a standard representation for `axis` you should use. Check the end of this section:

https://python-sprints.github.io/pandas/guide/pandas_docstring.html#section-3-parameters

Reply:
Just to be clear I'm asking for the preceding line to reflect the standard. The description down here can clarify the behavior of None since that is what differs for this method

Reply:
@WillAyd Thanks, I could rewrite similar to

```
axis : int, str, None
    {0 or ‘index’, 1 or ‘columns’, None}, default 0.
```

and change the two highlight functions the same way. My only worry would be that this less verbose version does not make it as clear as the previous wording that `'columns'` actually means row-wise. Maybe copying the wording from `apply` and `bar` would be a solution?

```
axis : int, str or None
    apply to each column (``axis=0`` or ``'index'``)
    or to each row (``axis=1`` or ``'columns'``) or
    to the entire DataFrame at once with ``axis=None``
```

Reply:
Sure - there's more freedom around the description so I think copying that over is fine. Just wanted to make sure we use the same type line as elsewhere as I think in the future we might LINT for that standard

Reply:
@WillAyd Updated. I made a few additional changes to unify the axis docstrings in this file and modified punctuation.